examples/chip-tool/commands/pairing/Commands.h

```diff
@@ -272,6 +273,7 @@ void registerCommandsPairing(Commands & commands, CredentialIssuerCommands * cre
         //        make_unique<CommissionedListCommand>(),
         make_unique<StartUdcServerCommand>(credsIssuerConfig),
         make_unique<OpenCommissioningWindowCommand>(credsIssuerConfig),
+        make_unique<OpenJointCommissioningWindowCommand>(credsIssuerConfig),
```

Comment:
Main goal of jf-control-app is to avoid changing chip-tool files: what jf-control-app did was to copy inside its own folder the chip-tool files that required specific JF changes. However maybe we don't have to copy the entire Command.h file and manage somehow to make this specific change directly in the main.cpp of the jf-control-app during command registration.

Reply:
That was the original implementation, but your earlier review requested this change. To avoid unnecessary back-and-forth, the original approach has been reinstated and is reflected in the latest commit. Let's avoid revisiting this unless there's a strong reason.

Reply:
Untrue:

1. First iteration of this PR: jcm open-joint-commissioning-window. My review: it's more natural to have the open-joint-commissioning-window inside the pairing command (e.g.: pairing open-joint-commissioning-window) because the existing open-commissioning-window follows the same pattern (e.g.: pairing open-commissioning-window command)
2. Second iteration of this PR: pairing open-joint-commissioning-window. My review: good invocation pattern but this required changes inside a chip-tool file (Commands.h) so I was wondering if there is a way to avoid changing Commands.h _while_ keeping the same invocation pattern
3. You reverted to the first iteration of this PR while suggesting that my requested changes are triggering unnecessary back-and-forth.